src/benchmarks/real-world/Roslyn/RoslynApis.cs

```diff
@@ -0,0 +1,26 @@
+
+using System.IO;
+using BenchmarkDotNet.Attributes;
+using Microsoft.CodeAnalysis;
+
+namespace CompilerBenchmarks
+{
+    /// <summary>
+    /// A collection of benchmarks for Roslyn APIs
+    /// </summary>
+    [BenchmarkCategory("Roslyn")]
+    public class RoslynApis
+    {
+        [Benchmark]
+        public object BuildAnalyzerConfigs()
```

Comment:
could you please return `ImmutableArray<AnalyzerConfigOptionsResult>` instead of an `object`? Casting struct to an object introduces boxing and adds some extra memory allocation which originally is not part of the measured APIs and should be avoided.

```suggestion
        public ImmutableArray<AnalyzerConfigOptionsResult> BuildAnalyzerConfigs()
```